lib/cartopy/examples/lines_and_polygons/always_circular_stereo.py

```diff
@@ -1,4 +1,8 @@
-__tags__ = ['Lines and polygons']
+"""
+Always Circular Stereo
```

Comment:
A bit weird for a title; it's more like "Keep Boundary Circular" or something along those lines.